Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "outpack"
-version = "0.3.2"
+version = "0.3.3"
```

Comment:
This is unrelated to this PR, but in the past I've done explicit PRs for bumping the version number, following https://github.com/mrc-ide/outpack_server?tab=readme-ov-file#releasing

I would be fine with using the same workflow we do in our R stuff, and bump the version on every commit. We'd want to tweak our CI to a) ensure the version has been bumped and b) push to PyPI on every push to main, not just on tags.